back/engines/free/email_campaigns/app/views/email_campaigns/internal_comment_on_idea_assigned_to_you_mailer/campaign_mail.mjml

```diff
@@ -0,0 +1,3 @@
+<%= render partial: 'email_campaigns/internal_comments/internal_comment_mail_body', locals: {internal_comment_body: event.internal_comment_body} %>
```

Comment:
```suggestion
<%= render 'email_campaigns/internal_comments/internal_comment_mail_body', internal_comment_body: event.internal_comment_body %>
```
It should be the same but slightly shorter. The same can be done in 2 other lines too.